### PR TITLE
Make references to the ASC less intimidating for new contributors

### PR DESCRIPTION
**Pull request by Walavouchey**
Part of #6793, but may close it depending on whether renaming the asc is deemed necessary. (there was no response on that point anywhere so I'm not doing that change in this pr.)

This does touch `CONTRIBUTING.md`, so I'll ping @peppy and @Ephemeralis. Read the linked issue for discussion and rationale.

Wording suggestions very welcome here.

(the small rewording and formatting fixes in the asc are slightly unrelated but I did commit those while at it)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by peppy — DISMISSED** [review]

File: wiki/Article_styling_criteria/en.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The article styling criteria (ASC) serve as the osu! wiki's enforced styling sta
 
 All articles should aim at having proper grammar, correct spelling, and correct information.
 
+These articles are primarily tools to aid in reviewing, and the information presented represents the consensus of osu! wiki contributors formed over the years. Since the wiki is a collaborative effort through the review process, it is not necessary to read or memorise all of the ASC at once. If you are looking to contribute, read the [contribution guide](/wiki/osu!_wiki/Contribution_guide).
```
Comment (TicClick) [inline wiki/Article_styling_criteria/en.md:14]: I know that you are referring to the writing and formatting guidelines slash rules, but the first sentence sounds like "these articles" from its beginning and "all articles" from the previous one are the same.

also, "... information _presented represents_ ..."

File: wiki/osu!_wiki/en.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Articles are composed according to the [styling criteria](/wiki/Article_styling_
 
 Despite having the *wiki* suffix in its name, the osu! wiki does not fully satisfy [the Wiki concept](https://en.wikipedia.org/wiki/Wiki):
 
-- Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is expected, which raises the threshold necessary for entry.
+- Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is necessary in order to contribute, which raises the threshold necessary for entry. The osu! wiki does however provide some quick guides in the [contribution guide](/wiki/osu!_wiki/Contribution_guide) in order to alleviate this issue.
```
Comment (TicClick) [inline wiki/osu!_wiki/en.md:34]: "... provides _guides_ in the contribution _guide_ ..."

**Review by TicClick — CHANGES_REQUESTED** [review]
minor things, looks good to me otherwise

File: wiki/Article_styling_criteria/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Article_styling_criteria/en.md:14]: a) fair point

b) mind giving an alternative? it sounds fine to me

File: wiki/osu!_wiki/en.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/osu!_wiki/en.md:34]: was thinking "tutorials" but thought that was a bit of a clunkier notion to what are more like quick-start guides than actual github tutorials

File: wiki/Article_styling_criteria/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Article_styling_criteria/en.md:14]: _These articles are primarily tools to aid in reviewing and represent the consensus of osu! wiki contributors formed over the years_

Comment (Walavouchey) [reply wiki/Article_styling_criteria/en.md]: Pushed a change - I think it became a bit more eloquent if I do say so myself

File: wiki/osu!_wiki/en.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Articles are composed according to the [styling criteria](/wiki/Article_styling_
 
 Despite having the *wiki* suffix in its name, the osu! wiki does not fully satisfy [the Wiki concept](https://en.wikipedia.org/wiki/Wiki):
 
-- Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is expected, which raises the threshold necessary for entry.
+- Improvement of existing articles and creation of new ones are encouraged, but basic knowledge of [GitHub](https://github.com) is necessary in order to contribute, which raises the threshold necessary for entry. The osu! wiki does however provide a [contribution guide](/wiki/osu!_wiki/Contribution_guide) to help new contributors get started.
 - All changes to the osu! wiki are pre-moderated and undergo a review process, where a contributor is expected to collaborate with the wiki maintainers and respond to feedback.
```
Comment (Walavouchey) [inline wiki/osu!_wiki/en.md:35]: collaborative effort

```suggestion
- All changes to the osu! wiki are pre-moderated and undergo a review process, where a contributor is expected to collaborate with reviewers and respond to feedback.
```

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**